front/app/components/ConsentManager/Banner.tsx

```diff
@@ -216,11 +217,12 @@ class Banner extends PureComponent<Props & InjectedIntlProps> {
         <CloseButton
           type="button"
           className="integration-button-close"
-          title={formatMessage(messages.ariaButtonClose)}
-          aria-label={formatMessage(messages.ariaButtonClose)}
           onClick={onAccept}
         >
           <CloseIcon name="close" />
+          <ScreenReaderOnly>
```

Comment:
I wonder if we shouldn't create a new component `IconButton` that requires these accessibility features, instead of having to add (and possibly forget them) every time.